utils/src/unit-test/java/uk/gov/moj/sdt/utils/AbstractSdtXmlTestBase.java

```diff
@@ -304,22 +301,16 @@ protected void validateXsd(final String xmlPathname, final String xsdPathname, f
         List<String> remainingExpectedErrors = null;
         try {
             remainingExpectedErrors = evaluateXsd(xmlPathname, xsdPathname, errorFilePathname);
-        } catch (final IOException e) {
-            LOGGER.error("Exception while validating XML [" + xmlPathname + "] with XSD [" + xsdPathname + "]", e);
-            Assert.fail("Exception while validating XML [" + xmlPathname + "] with XSD [" + xsdPathname + "]");
-        } catch (final SAXException e) {
-            LOGGER.error("Exception while validating XML [" + xmlPathname + "] with XSD [" + xsdPathname +
-                    "] on line " + ((SAXParseException) e).getLineNumber(), e);
-            Assert.fail("Exception while validating XML [" + xmlPathname + "] with XSD [" + xsdPathname + "]");
-        } catch (final ParserConfigurationException e) {
-            LOGGER.error("Exception while validating XML [" + xmlPathname + "] with XSD [" + xsdPathname + "]", e);
-            Assert.fail("Exception while validating XML [" + xmlPathname + "] with XSD [" + xsdPathname + "]");
+        } catch (final IOException|SAXException|ParserConfigurationException e) {
+            LOGGER.error("Exception while validating XML [{}] with XSD [{}]",
+                    xmlPathname, xsdPathname, e);
+            fail("Exception while validating XML [" + xmlPathname + "] with XSD [" + xsdPathname + "]");
```

Comment:
Looks like the SAXParseException needs to be separated out.  As slightly different error message was displayed for it previously - one that contained the line number on which the error occurred, i.e.

`LOGGER.error("Exception while validating XML [" + xmlPathname + "] with XSD [" + xsdPathname + "] on line " + ((SAXParseException) e).getLineNumber(), e);`